cmd/incus/action.go

```diff
@@ -331,7 +331,12 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 	err = cli.CancelableWait(op, &progress)
 	if err != nil {
 		progress.Done("")
-		return fmt.Errorf("%s\n"+i18n.G("Try `incus info --show-log %s` for more info"), err, nameArg)
+		projectArg := ""
+		if conf.ProjectOverride != "" && conf.ProjectOverride != "default" {
+			projectArg = " --project " + conf.ProjectOverride
+		}
```

Comment:
I'd use `api.ProjectDefaultName` instead of the hardcoded `"default"` in here.

Reply:
Noted. FYI, `"default"` is hardcoded in many places in `cmd/`.

Reply:
Yeah, it's one of those where we're trying to not introduce new ones at least :)